### ISSUE TITLE
Oxygen Woocommerce Integration is bloated

### ISSUE THREAD

**Issue by madjedo** [soflyy/oxygen-bugs-and-features#1970]
**Describe the feature you'd like to see included in Oxygen.**
Using the Woocommerce integration. You also get ALL of woocommerce css loaded on every page inside universal.css. (Mind you, this is not just a few lines of code).
The woocommerce css output should be limited to product pages/shop/archive/cart/checkout + user chosen custom pages.

**What are the use cases for this feature?**
Fast site, no bloat. SEO, Pagespeed scores, usability etc.

**Examples of this feature or functionality.**
N/A

**Comment by samjco** [comment]
Can we disable Woocommerce styles?

**Comment by madjedo** [comment]
Another thing I have noticed is that Oxygen Builder is declaring all kinds of classes related to WooCommerce, on top of the ones already standard with WooCommerce. This adds even more bloat.

I don't need these classes declared at all, as I would like a pure and clean install as possible that's for me to style. So another feature to add is the one similar to the one you have already for standard Oxygen Builder, to let user install a "clean install" of your WooCommerce integration.

What's happening now is. WooCommerce have classes declared, Oxygen Builder declares classes and then I start to declare classes on top of those two. Basically the definition of bloat.

**Comment by cstrouse** [comment]
You can unload the assets you don't want in a custom functionality plugin.

**Comment by madjedo** [comment]
> You can unload the assets you don't want in a custom functionality plugin.

True, but in front-end, Oxygen Woocommerce styles are baked in with everything else _(/wp-content/plugins/oxygen/component-framework/oxygen.css)_. 

So in this scenario, I have to unload everything. They should be separate.

**Comment by skrypt-nl** [comment]
This should really be implemented asap. It's unnecessary bloat and can also be interfering with existing CSS on the website.
